_maps/RandomZLevels/museum.dmm

```diff
@@ -518,17 +518,9 @@
 /turf/open/indestructible/plating,
 /area/awaymission/museum)
 "ek" = (
-/obj/effect/decal/cleanable/crayon{
-	icon_state = "l";
-	pixel_x = -10
-	},
-/obj/effect/decal/cleanable/crayon{
-	icon_state = "i"
-	},
-/obj/effect/decal/cleanable/crayon{
-	icon_state = "e";
-	pixel_x = 12
-	},
+/obj/effect/decal/cleanable/crayon/lie1,
+/obj/effect/decal/cleanable/crayon/lie2,
```

Comment:
the pixel offsets are gone

Reply:
nvm theyre on the object by default but thats
pretty stupid

Reply:
There's nothing wrong with that, pixel offsets are not the devil